### PR TITLE
Pullrequest Android 7 (Nougat) notification permissions

### PR DESCRIPTION
**Pull request by BenSartor**

### CODE REVIEW

File: app/src/main/res/values-de/strings.xml [hunk 1]
```diff
@@ -39,6 +39,7 @@
     <string name="main_activity_menu_delete_account">Einstellungen löschen</string>
     <string name="main_activity_menu_fake_telephone_book">Screenshot Telefonbuch</string>
     <string name="main_activity_menu_fake_telephone_book_disable">Normales Telefonbuch</string>
+    <string name="main_activity_menu_notification_settings">Nicht stören Einstellungen</string>
```
[the remark this answers is not in the document]
Comment (BenSartor) [reply app/src/main/res/values-de/strings.xml]: "Nicht stören"-Einstellungen?

[the remark this answers is not in the document]
Comment (jritzerfeld) [reply app/src/main/res/values-de/strings.xml]: We've never used quotation marks, yet.

[the remark this answers is not in the document]
Comment (BenSartor) [reply app/src/main/res/values-de/strings.xml]: Yeah, but the Do Not Disturb settings is the first place I see android using quotation marks. But only in german as far a as I have checked.

File: app/src/main/res/values-de/strings.xml [hunk 2]
```diff
@@ -141,7 +141,7 @@
     <!-- Zugriff während "Nicht stören" -->
     <!-- Zugriff auf "Nicht stören" für Simlar zu lassen?-->
     <!-- Die App kann "Nicht stören" aktivieren/deaktivieren und Änderungen an verwandten Einstellungen vornehmen?-->
-    <string name="permission_explain_text_notification_policy_access">Simlar möchte wöhrend Telefonaten den Nicht stören Modus aktivieren, damit z.B: weitere Anrufe nicht vibrieren oder laut klingeln.</string>
+    <string name="permission_explain_text_notification_policy_access">Simlar möchte während Telefonaten den \"Nicht stören\"-Modus aktivieren, damit z.B: weitere Anrufe nicht vibrieren oder laut klingeln.</string>
```
Comment (jritzerfeld) [inline app/src/main/res/values-de/strings.xml]: '.' missing after "z.B" still missing.

**Pull request merged**